docroot/modules/custom/prisoner_hub_sub_terms/README.md

```diff
@@ -12,21 +12,23 @@ The results will be a paginated list of series and sub-categories.
 - Any sub category of the {uuid} (only direct sub-categories are included, not further levels)
 - Any series assigned to the {uuid} category (series assigned to sub-categories are not included).
 
-The results will be ordered by the most recently updated content within each sub term.
+The results will be ordered by the most recently created content within each sub term.
 When checking for the most recently updated content, we look at all levels of the taxonomy.  So changes to content that
-is in a sub-category multiple levels down, will affect the sorting order of it's parent sub-category.
+are in a sub-category multiple levels down, will affect the sorting order of it's parent sub-category (although this
+can take up to 24 hours to appear, see the cache tags info below).
 
 Note that if using a prison context, this again will be applied to the results.  So only content available in the
 current prison will be used for sorting.
 
-## Custom cachetags
-This module uses custom cachetags to invalidate the response when content has been updated or created.
-The cachetag looks like:
+## Custom cache tags
+This module uses custom cache tags to invalidate the response when content has been created.
+The cache tag looks like:
```

Comment:
I'm updating any reference to "cachetag" to be "cache tag", as this is the correct way to write it.